build.sbt

```diff
@@ -2474,6 +2475,7 @@ lazy val `engine-runner` = project
   .dependsOn(`library-manager`)
   .dependsOn(`language-server`)
   .dependsOn(`edition-updater`)
+  .dependsOn(`runtime-parser`)
```

Comment:
I had to update licenses in da144be1efae4d246bd7bb029fb65992ab6cfa86 - is it a result of this change!? How can an addition of a dependency to `engine-runner` remove licenses from `engine`, @radeusgd?

I'd be glad to have less dependencies in `runtime.jar` (and there are no `errorprone`, `guava` or `j2objc` classes in `runtime.jar`), but it is a bit surprising consequence.

Reply:
I'm confused as well to be honest.